setup.py

```diff
@@ -50,13 +50,14 @@
     # filenames to .c filenames, and we probably don't have the .c files.
     sys.path.insert(0, pjoin(dirname(__file__), 'fake_pyrex'))
     # Set setuptools extra arguments
+    # We need nosetests for numpy.testing<=1.15
     extra_setuptools_args = dict(
-        tests_require=['nose'],
+        tests_require=['pytest', 'nose'],
```

Comment:
Same question here: we do still need both nose and pytest?

Reply:
Same answer: Unfortunately yes...  Until we have Numpy 1.15 as a minimal version.  `numpy.testing` depends on `nose` until that version

Reply:
Gotcha. Yeah - I remember we discussed this before. Seems fine for now.